.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "printWidth": 80,
```

Comment:
This is a personal preference, but I like the TOML config more than this JSON-ish format. Prettier supports both.